library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeUnifiedStateImpl.java

```diff
@@ -40,6 +40,7 @@
 import com.google.common.collect.Sets;
 import com.google.common.util.concurrent.Futures;
 
+import com.datatorrent.api.Context;
```

Comment:
Can you please call out in the class level javadoc how this class differs from ManagedTimeStateImpl?